docs/pages/technical/registry.rst

```diff
@@ -67,5 +70,19 @@ RegisterEndpoint        Organization  ``Event.Payload.Organization == Certificat
     The Nuts Foundation will act as Root Certificate Authority so that intermediates are issued by an entity which is trusted
     by all participating parties. However, this Root Certificate Authority isn't operational at the time of writing so
     vendors are expected to self-sign their own CA certificates in the meantime.
-    This means when validating a *RegisterVendor* event the certificate which signed the JWS will be self-signed and
-    thus can't be validated. **This is the only case** where an unvalidated certificate should be added to the trust store.
\ No newline at end of file
+    This means when validating a ``RegisterVendor`` event the certificate which signed the JWS will be self-signed and
+    thus can't be validated. **This is the only case** where an unvalidated certificate should be added to the trust store.
+
+Versioning
+**********
+
+Events have a ``Version`` field indicating the version of the data structure. New versions might introduce new fields or
+change the datatype of existing fields (although this change must be backwards compatible.
+
+===========  ==================================================
+Version      Change
+===========  ==================================================
+0            Any version before introduction of ``Version``
+1            ``Version``, ``Ref`` and ``Prev`` added
```

Comment:
But without canonicalization this may cause lots of errors?

Reply:
This indeed causes problems for signing. Made an issue which must be fixed for v0.14: #95